### PR TITLE
Comments and search refactor

### PR DESCRIPTION
**Pull request by smile-eh**
Cleaned up the search file by moving some sections into their own components to make code cleaner and easier to understand.

### CODE REVIEW

**Comment by smile-eh** [comment]
Linked to task: CU-1wnwqfk

**Comment by Emilie-norman-fortin** [comment]
Task linked: [CU-1wnwqfk Improve documentation/documentation of front end, and refactor large classes](https://app.clickup.com/t/1wnwqfk)

File: front-end/src/components/search/QueryBox.js [hunk 1]
```diff
@@ -44,8 +35,7 @@ const QueryBox = (props) => {
 
     fetch(requestURL + new URLSearchParams({
         q:props.searchQuery,
-        df:`text_${props.currentLang}_txt`,
-        'q.op': "AND"
```
Comment (handshape) [inline front-end/src/components/search/QueryBox.js:48]: This looks like it reverts CU-21prurv which got committed in PR #24

File: front-end/src/components/search/result/Information.js [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+*/
+
+const Information = (props) =>{
+
+  const submitSparqlRelated = async(id) => {
+    /* Single search term: https://dev.handshape.com/sparql?query=SELECT%20*%20{%3Chttps://www.canada.ca/en/privy-council/ext/statutory-instrument/P-15.6%3E%20%3Chttps://schema.org/url%3E%20?o}%20LIMIT%208&Accept=application/sparql-results%2Bjson */
+    const protocol = "https://";
+    //const currentHost = window.location.hostname;
+    const currentHost = "dev.handshape.com";
+    const spaqrlPath = "/sparql?"
```
Comment (handshape) [inline front-end/src/components/search/result/Information.js:21]: We'll need to address this in CU-1wnv7nv

File: front-end/src/components/search/result/Information.js [hunk 3]
```diff
@@ -0,0 +1,179 @@
+import React from "react";
```
Comment (handshape) [inline front-end/src/components/search/result/Information.js:1]: This whole file is more than comments and refactoring; it's entirely new. It looks like work-in-progress, as the server response appears to be mocked...

**Comment by handshape** [comment]
Aside from the reverting change in QueryBox.js, this looks good to go.

File: front-end/src/components/search/QueryBox.js (hunk 1 above)
Comment (handshape) [reply front-end/src/components/search/QueryBox.js:48]: Resolved this one myself. This was the only blocker for merging, so I'm proceeding.

**Pull request merged**